doc/Language/regexes.pod6

```diff
@@ -1255,6 +1255,18 @@ list of predefined subrules is listed in
 L<S05-regex|https://design.perl6.org/S05.html#Predefined_Subrules> of design
 documents.
 
+=head1 X«Regex Interpolation|Regex Interpolation; <$patten>»
+
+If you want to build a regex using pattern given at the runtime, regex
+interpolation is what you looking for.
+
+    sub runtime-regex(Str $patten --> Regex) { /<$patten>/ }
+
+    my $regex = runtime-regex(get);  # INPUT  : «\d+␤»
+    my $text = get;                  # INPUT  : «42␤»
+
+    say $text ~~ $regex;             # OUTPUT : «｢42｣␤»
```

Comment:
Perhaps don't use a space before the column (“OUTPUT:” instead of “OUTPUT :”). I think the idea was that this should be parseable so that we can use it to check the output, so better keep it consistent with other things. Also, normally you can copy-paste the output from ``camelia`` bot directly.